test/test_cmdstan_args.py

```diff
@@ -103,88 +103,31 @@ def test_good() -> None:
     assert 'algorithm=hmc engine=nuts' not in ' '.join(cmd)
 
 
-def test_bad() -> None:
-    args = SamplerArgs(iter_warmup=-10)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(iter_warmup=10, adapt_engaged=False)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(iter_sampling=-10)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(thin=-10)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(max_treedepth=-10)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(step_size=-10)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(step_size=[1.0, 1.1])
-    with pytest.raises(ValueError):
-        args.validate(chains=1)
-
-    args = SamplerArgs(step_size=[1.0, -1.1])
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_delta=1.1)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_delta=-0.1)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(max_treedepth=12, fixed_param=True)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(metric='dense', fixed_param=True)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(step_size=0.5, fixed_param=True)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_delta=0.88, adapt_engaged=False)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_init_phase=0.88)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_metric_window=0.88)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_step_size=0.88)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_init_phase=-1)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_metric_window=-2)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_step_size=-3)
-    with pytest.raises(ValueError):
-        args.validate(chains=2)
-
-    args = SamplerArgs(adapt_delta=0.88, fixed_param=True)
+@pytest.mark.parametrize("kwargs", [
```

Comment:
Sorry, just noticed that this change snuck in from #639. I can rewrite the history to get rid of this change if preferable.